index.html

```diff
@@ -12,74 +12,165 @@
+                </a>
+            </li>
+            <li>
+                <a href="#" rel="noopener noreferrer nofollow" target="_blank">
+                    <img src="img/Vector1.png" alt="">
+                </a>
+            </li>
+        </ul>
+        <img class="work" src="img/Element-Right.png" alt="">
+    </div>
+
+    <!-- ----SECOND SECTION START -------->
+
+    <h2 class="work-titl">MY RECENT WORK</h2>
     <img class="hr" src="img/Vector-4.png" alt="hrl">
-    <section class="skils-container">
-        <div class="cont">
-            <span>language</span>
-            <ul>
-                <li>Javascript</li>
-                <li>Ruby</li>
-                <li>Html</li>
-                <li>Css</li>
-            </ul>
-        </div>
-        <div class="cont">
-            <span>Frameworks</span>
-            <ul class="list">
-                <li>Bootstrap</li>
-                <li>Ruby On Rials</li>
-                <li>RSPec</li>
-                <li>CapyBara</li>
-                <li>Selenium</li>
-            </ul>
-        </div>
-        <div class="cont">
-            <span>Skils</span>
-            <ul class="list">
-                <li>Codekit</li>
-                <li>Github</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-            </ul>
-        </div>
+
+    <section class="second-container">
+        <article class="card-container">
+            <img src="img/Snapshoot-Portfolio.png" alt="img-placeholder">
+            <h3 class="work-titl-2">Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                text.
+            </p>
+            <div class="list-container">
+                <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li class="ruby">Ruby</li>
+                </ul>
+            </div>
+            <button type="button" class="btn">See Project</button>
+        </article>
+        <article class="card-container">
+            <img src="img/Snapshoot-Portfolio(1).png" alt="img-placeholder">
+            <h3 class="work-titl-2">Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                text.
+            </p>
+            <div class="list-container">
+                <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li class="ruby">Ruby</li>
+                </ul>
+            </div>
+            <button type="button" class="btn">See Project</button>
```

Comment:
- [ ] [OPTIONAL] Good job so far :smile: but please follow the button styling: Active, Hover and Pressed  as defined on Figma design :smile: 
![Screenshot from 2022-08-24 15-40-58](https://user-images.githubusercontent.com/6615394/186462786-29b7aec2-0274-446c-af60-f2e117e1ec0d.png)